src/OAuth1/Provider/Atlassian.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry https://github.com/ovr <talk@dmtry.me>
+ */
+
+namespace SocialConnect\OAuth1\Provider;
+
+use SocialConnect\Common\Http\Client\Client;
+use SocialConnect\Common\Http\Client\ClientInterface;
+use SocialConnect\OAuth1\Signature\MethodRSASHA1;
+use SocialConnect\Provider\AccessTokenInterface;
+use SocialConnect\Provider\Consumer;
+use SocialConnect\Provider\Exception\InvalidResponse;
+use SocialConnect\OAuth1\AbstractProvider;
+use SocialConnect\Common\Entity\User;
+use SocialConnect\Common\Hydrator\ObjectMap;
+use SocialConnect\Provider\Session\SessionInterface;
+
+class Atlassian extends AbstractProvider
+{
+    private $baseUri;
```

Comment:
blank line after `private $baseUri;`
and please doc block that it's a string
Thank